src/main/java/com/revature/ams/util/auth/AuthService.java

```diff
@@ -0,0 +1,23 @@
+package com.revature.ams.util.auth;
+
+import com.revature.ams.Member.Member;
+import com.revature.ams.Member.MemberService;
+
+import javax.security.sasl.AuthenticationException;
+
+// TODO: REVIEW ME
+public class AuthService {
+    private final MemberService memberService; // Assuming a MemberService instance
+
+    public AuthService(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    public Member login(String email, String password) throws AuthenticationException {
+        Member member = memberService.findByEmailAndPassword(email, password);
+        if(member == null) throw new AuthenticationException("Invalid member credentials, please try again");
```

Comment:
What is AuthenticationException, and what does it really do here besides just being an exception to throw? Does it do anything special?

Reply:
Nothing special, just an already existing exception to throw that clearly identifies what's going wrong